### ISSUE TITLE
Works perfectly in local but when deployed to external tomcat server returns 404

### ISSUE THREAD

**Issue by 0VirtualNomad0** [danvega/jwt-username-password#4]
Hi,
The video tutorial and the latest code changes work perfectly locally.
But when I deploy it to an external server shows no server error but returns 404 for every request.
Is there another setting we need to do if we want to deploy it on an external server?

**Comment by danvega** [comment]
No. How are you packaging the application? Is there any more info in the logs?
